starlite/middleware/exceptions/middleware.py

```diff
@@ -47,6 +47,8 @@ async def __call__(self, scope: "Scope", receive: "Receive", send: "Send") -> No
             await self.app(scope, receive, send)
         except Exception as e:  # pylint: disable=broad-except
             starlite_app = scope["app"]
+            if self.debug and (logger := starlite_app.logger):
+                logger.debug("Exception in ASGI application", exc_info=True)
```

Comment:
I suggest to change this message: 

```suggestion
                logger.debug("exception raised for request to route %s", scope["path"], exc_info=True)
```

Maybe add more details from the scope context as well. Also - lets use lower case messages, which we use in the exception messages we raise as well.

Reply:
note: https://medium.com/swlh/why-it-matters-how-you-log-in-python-1a1085851205